src/Cosmonaut/CosmosStoreSettings.cs

```diff
@@ -74,7 +78,8 @@ public class CosmosStoreSettings
             string authKey,
             ConnectionPolicy connectionPolicy = null,
             IndexingPolicy indexingPolicy = null,
-            int defaultCollectionThroughput = CosmosConstants.MinimumCosmosThroughput)
+            int defaultCollectionThroughput = CosmosConstants.MinimumCosmosThroughput,
+            UniqueKeyPolicy uniqueKeyPolicy = null)
```

Comment:
Can you remove that from the ctor? I am moving away from optional parameters via the ctor